src/modules/IscsiClientLib.rb

```diff
@@ -238,7 +235,7 @@ def getiBFT
           return @ibft
         end
         ret = SCR.Execute(path(".target.bash_output"),
-                          "lsmod |grep -q iscsi_ibft || modprobe iscsi_ibft")
+          "lsmod |grep -q iscsi_ibft || modprobe iscsi_ibft")
```

Comment:
Out of scope, but this looks a bit strange to me, the `man modprobe` says

```
Normally, modprobe will succeed (and do nothing) if told to insert
a module which is already present
```

So probably just simple `modprobe iscsi_ibft` should be enough here. But you can leave the current implementation, that's out of Rubocop scope, I just noticed this when seeing the diff...

Reply:
Yes, but this PR is only about rubocop auto-generated changes.